packages/storybook/cxl-ui/cxl-app-layout/layout=2c-l.stories.js

```diff
@@ -14,7 +15,10 @@ export default {
 };
 
 export const CXLAppLayout2cl = () => {
-  const playbookId = 1234;
+  const playbookId = number('Playbook ID', 1234);
+  const userId = number('User ID', 5678);
+  const playbookSaved = boolean('Is playbook saved?', false);
+  const saveUnsaveTextVisible = boolean('Save/unsave text visible?', true);
```

Comment:
What is this toggle useful for? Feels like a drop candidate.

Reply:
you mean booleans?

Reply:
> you mean booleans?

Specifically saveUnsaveTextVisible. Others purpose looka good.

Reply:
` boolean('Save/unsave text visible?', true); ` this is for showing the possibility to use the component with just a star visible, which can be used in the top-right corner of the card (dashboard), but probably this should be shown in the component's own story, I can do one if you confirm there's need for it.